ospkg/sign_test.go

```diff
@@ -0,0 +1,11 @@
+package ospkg_test
```

Comment:
Why is it a different package?

Reply:
See above

Reply:
Yes I agree, checked the docs and they also don't explicitly recommend using the `_test` package name. Gonna revert this change then and disable the linting test that checks those.